dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+
+set -e
+
+init() {
+#OS specific defaults
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    DEFAULT_OC_PUBLIC_HOSTNAME="192.168.65.2"
+    DEFAULT_OC_PUBLIC_IP="192.168.65.2"
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTID" ] && export CHE_OAUTH_GITHUB_CLIENTID=d210c15f1a0c026a808c;
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTSECRET" ] && export CHE_OAUTH_GITHUB_CLIENTSECRET=cc2589e6eed89ca460c8dbd13b9342cfddbe8098;
+else
+    DEFAULT_OC_PUBLIC_HOSTNAME="127.0.0.1"
```

Comment:
Is this value valid for a Linux? If no (it doesn't work for me) then I suggest to fail script if env var is not set externally

Reply:
that is more placeholder for linux we have to use real IP.
I think I can get that IP from che-ip tool

Reply:
done